tutorials/abap-environment-amdp-profiling/abap-environment-amdp-profiling.md

```diff
@@ -494,6 +494,12 @@ The **Trace Results** view appears. The **ABAP Managed Database Procedures** tab
 
 ![Image depicting step12d-amdp-tab](step12d-amdp-tab.png)
 
+Note: In the ABAP Profiler display for the AMDP Trace, the column "Execution Time" sometimes contains values that are larger than the values in column "Duration". As the ABAP Profiler usually displays elapsed time, the column "Execution Time" may only be smaller than "Duration", as there might be some "Compile Time" involved in the overall duration.
```

Comment:
Can you code this as a note? ( >In the ABAP ... )

Also, can you remove the word "Note:" from the note?